### DISCUSSION TITLE
How to...

### DISCUSSION THREAD

**Discussion by vjagodnik** [Future-House/paper-qa#267] [Q&A] [answered]
Hello to all,
Is there a working "how to..." for starting the use of this package because no matter what I do or additional package I install it seems that it is not working for me.
Thank you
v

**Comment by jamesbraza** [comment, marked as answer]
Hi @vjagodnik yes we have added a ton to the `README` in the past weeks, and released version 5 (which is a major refactor of everything). Feel free to reopen a new issue/discussion if you're still stuck on stuff
